tools/validators/instance_validator/tests/fake_instances/GOOD/translation_req_field_marked_missing.yaml

```diff
@@ -15,7 +15,7 @@
 SDC_EXT-17-GUID:
   type: HVAC/SDC_EXT
   code: SDC_EXT-17
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.